systemd/url-bot-rs.service

```diff
@@ -0,0 +1,12 @@
+[Unit]
+Description=url-bot-rs
+After=network.target
+Requires=network.target
+
+[Service]
```

Comment:
Probably want to enforce the use of a `url-bot-rs` user, and perhaps even the hardening flags commented out here: 
https://limnoria.readthedocs.io/en/latest/use/supybot-botchk.html#systemd-service

Reply:
I've added running as `url-bot-rs` user in 394f883, and an issue for adding hardening options as issue #89

Reply:
DynamicUser seems dodgy from what i've read, a hard-coded user seems more reliable.